mitodl/open-discussions
# add level and department to search

#### Pre-Flight checklist

#### What are the relevant tickets?
closes https://github.com/mitodl/open-discussions/issues/3139

#### What's this PR do?
Adds level and department to the course elasticsearch index

#### How should this be manually tested?
Update ocw course data with `docker-compose  run web python manage.py backpopulate_ocw_data --overwrite --skip-s3`
Recreate the elasticsearch index with `docker-compose  run web python manage.py recreate_index`

Verify that department is populated for OCW courses in your local database. Verify that department and level are in `http://localhost:9101/discussions_local_course_default/_search`
